dartagnan/src/main/java/com/dat3m/dartagnan/parsers/program/ParserInlineAsm.java

```diff
@@ -0,0 +1,46 @@
+package com.dat3m.dartagnan.parsers.program;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.antlr.v4.runtime.CharStream;
+import org.antlr.v4.runtime.CommonTokenStream;
+import org.antlr.v4.runtime.ParserRuleContext;
+
+import com.dat3m.dartagnan.exception.ParsingException;
+import com.dat3m.dartagnan.exception.UnrecognizedTokenListener;
+import com.dat3m.dartagnan.expression.Expression;
+import com.dat3m.dartagnan.parsers.InlineAsmLexer;
+import com.dat3m.dartagnan.parsers.InlineAsmParser;
+import com.dat3m.dartagnan.parsers.program.visitors.VisitorInlineAsm;
+import com.dat3m.dartagnan.program.Function;
+import com.dat3m.dartagnan.program.Register;
+import com.dat3m.dartagnan.program.event.Event;
+
+public class ParserInlineAsm {
+
+    private VisitorInlineAsm visitor;
```

Comment:
This can be made a local variable. Alternatively, you initialize it in the constructor and get rid of the other three members.